### PR TITLE
[WIP] Append timestamp after dash and parse it out

### PR DESCRIPTION
**Pull request by mkpankov**
Avoid adding on multiple timestamps

Signed-off-by: Michael Pankov <work@michaelpankov.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1522)

### CODE REVIEW

File: vagrant/scripts/install_iml_local.sh [hunk 1]
```diff
@@ -19,8 +19,8 @@ rpmdev-bumpspec python-iml-manager.spec
 # causes a re-install
 TS=$(date +%s)
 
-V=$(rpmspec -q --srpm --queryformat='%{VERSION}' /integrated-manager-for-lustre/rust-iml.spec)
-rpmdev-bumpspec -n $V.$TS rust-iml.spec
+V=$(rpmspec -q --srpm --queryformat='%{VERSION}' /integrated-manager-for-lustre/rust-iml.spec | cut -d - -f 1)
+rpmdev-bumpspec -n $V-$TS rust-iml.spec
```
Comment (nlinker) [inline vagrant/scripts/install_iml_local.sh]: Does this dot separator interfere with other stuff, so you've decided to change it?

**Comment by mkpankov** [comment]
Don't merge, investigating a failure

**Comment by mkpankov** [comment]
Doesn't work properly because there's `-1` appended by something afterwards, and our version is considered older because of it.

File: vagrant/scripts/install_iml_local.sh (hunk 1 above)
Comment (ip1981) [reply vagrant/scripts/install_iml_local.sh]: Possible to copy the file and work with the copy?

Comment (mkpankov) [reply vagrant/scripts/install_iml_local.sh]: https://github.com/whamcloud/Dockerfiles/issues/43

**Pull request closed without merging**